setup/ansible-inventory

```diff
@@ -77,10 +77,6 @@ iojs-release-armv7-wheezy-1
 release-scaleway-debian7-arm-1
 
 [iojs-raspbian]
-test-requireio_joeyvandijk-debian7-arm_pi2-1
-test-requireio_svincent-debian7-arm_pi2-2
```

Comment:
Yeah, sorry, that's fixing something that made it in last time that shouldn't. I reprovisioned those two machines so pulled them into their own `[iojs-raspbian]` section and renamed the old one to `[xiojs-raspbian]` so I could run the playbook without `--limit`. It was a mistake and this commit corrects it, including getting rid of the `[xiojs-raspbian]` header. Those two machines are still in the mix, lower down.